purchase_order_supplier_return/models/stock_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _action_done(self, cancel_backorder=False):
+        todo_moves = super()._action_done(cancel_backorder=cancel_backorder)
```

Comment:
```suggestion
        done_moves = super()._action_done(cancel_backorder=cancel_backorder)
```
I follow the remark by JE on one of my PRs :wink: